cardano-api/src/Cardano/Api.hs

```diff
@@ -10,6 +11,8 @@ module Cardano.Api
   , KeyPair (..)
   , Network (..)
   , PublicKey (..)
+  , ShelleyKeyDiscriminator (..)
+  , ShelleyVerificationKey (..)
```

Comment:
Its a pity that these leak from the API, but lets get the API finished and then see if we can clean it up.